pkg/ccl/roleccl/role.go

```diff
@@ -95,6 +96,12 @@ func grantRolePlanHook(
 		return nil, nil, nil, false, nil
 	}
 
+	if grant.AdminOption {
+		sqltelemetry.IncIAMGrant(true)
```

Comment:
can't you pass `grant.AdminOption` into the function here, and avoid the branch?

Reply:
Oops yeah you're right